routers/web/org/setting.go

```diff
@@ -89,7 +89,6 @@ func SettingsPost(ctx *context.Context) {
 		// reset ctx.org.OrgLink with new name
 		ctx.Org.OrgLink = setting.AppSubURL + "/org/" + url.PathEscape(form.Name)
```

Comment:
Please be care ful with `url.PathEscape` in special case when url is already html encoded.